### PR TITLE
Placeholder in IE now will display as placeholder not a default value

### PR DESCRIPTION
**Pull request by YaroslavOvdii**
@sofiiakvasnevska 
## Issue
https://github.com/Fliplet/fliplet-studio/issues/5060

## Description
Placeholder in IE now will display as a placeholder, not a default value

## Backward compatibility

This change is fully backward compatible.

## Notes
Same as in this [PR](https://github.com/Fliplet/fliplet-widget-slider/pull/19)

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
-﻿.idea/*
\ No newline at end of file
+﻿.idea/*
+.history
```
Comment (tonytlwu) [inline .gitignore:2]: Why is this change necessary?

Comment (YaroslavOvdii) [reply .gitignore:2]: @tonytlwu 
I using local history module for the Visual Studio Code, which saving change history locally into `.history` folder. To prevent adding these changes to the repository I add this line.

Comment (tonytlwu) [reply .gitignore:2]: @squallstar I don't see this change in any of the other similar PRs. Are you happy with this?

Comment (squallstar) [reply .gitignore:2]: @tonytlwu yep that's fine with me, I have that plugin too installed.

**Review by squallstar — APPROVED** [review]
@sofiiakvasnevska this is ready to be deployed on **staging** for QA.

- **Risk**: Low
- **Affected**: Widget interface on IE
- **Verify**: Verify the issue is resolved
